recipe/meta.yaml

```
@@ -69,7 +71,7 @@ requirements:
     - cloudpickle >=0.2.1
     - _openmp_mutex  # [linux]
   run_constrained:
-    - matplotlib-base >=3.0.3
+    - matplotlib-base >=3.3.0
     - pooch >=1.3.0
```

```suggestion
    - pooch >=1.3.0
    - astropy >=3.1.2
```
https://github.com/scikit-image/scikit-image/blob/v0.20.0/pyproject.toml#L107